API/hermes_abi/hermes_abi.h

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#ifndef HERMES_ABI_HERMES_ABI_H
+#define HERMES_ABI_HERMES_ABI_H
+
+#include <assert.h>
+#include <stdalign.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#ifndef HERMES_EXPORT
```

Comment:
It is better to optimize the header file for users, not for the implementation.
We better reverse the logic here and not to define the HERMES_EXPORT unless we have another definition that says that we are implementing the API rather than consuming it.

Reply:
This was only used in one place at the bottom of this header, so I've actually removed the definition for now.